app/src/components/footerV2.js

```diff
@@ -2,55 +2,90 @@ import React from "react";
 import LogoFr from "../assets/fr.png";
 import LinkTo from "../assets/icons/LinkTo";
 import useDevice from "../hooks/useDevice";
-import { useLocation } from "react-router-dom";
 import SNU from "../assets/logo-snu.png";
 
-const Footer = () => {
-  const { pathname } = useLocation();
-  const desktop = useDevice() === "desktop";
+const Footer = ({ marginBottom }) => {
   const mobile = useDevice() === "mobile";
-  const smallButton = !["/preinscription/sejour"].includes(pathname);
-  const largeButton = ["/preinscription/done"].includes(pathname);
-  const noButton = ["auth"].findIndex((route) => location.pathname.includes(route)) !== -1;
 
-  return (
-    <div
-      className={`${desktop || noButton ? "mb-0" : largeButton ? "mb-[20vh]" : smallButton ? "mb-[12vh]" : "mb-0"} ${
-        desktop ? " border-t-[#000091] border-t pt-3 px-3" : mobile && "border-t border-t-[#E5E5E5]"
-      } w-full bg-white  pb-3`}>
-      <div className={`${desktop ? "flex justify-between w-full px-[108px]" : "px-3"} text-sm text-[#3A3A3A] py-3`}>
-        <div className={`${desktop && "basis-[50%]"} mb-4 flex`}>
+  return mobile ? (
+    <div className={`mb-[${marginBottom}] border-t border-t-[#E5E5E5] w-full bg-white  pb-3`}>
```

Comment:
```suggestion
    <div className={`${marginBottom ? mb-[${marginBottom} ] : ""} border-t border-t-[#E5E5E5] w-full bg-white  pb-3`}>
```
Je ferais juste un truc comme ca ! Pas beosin de passer de parametre si on a rien